src/bsp/platform/stm32f4/xi_bsp_rng_stm32f4.c

```diff
@@ -6,18 +6,54 @@
 
 #include <xi_bsp_rng.h>
 
+#if !defined( XI_TLS_LIB_WOLFSSL )
+
+void xi_bsp_rng_init()
+{
+}
+
+uint32_t xi_bsp_rng_get()
+{
+    uint32_t random = 0;
+
+    return random;
+}
+
+void xi_bsp_rng_shutdown()
+{
+}
+
+#elif defined( XI_TLS_LIB_WOLFSSL ) /* WOLFSSL version of RNG implementation */
+
+#include <cyassl/ctaocrypt/random.h>
```

Comment:
great you've implemented wolfSSL's solution. Is it "perfect" or do we have to massage it more?
Also could you use wolfssl_ prefixes instead of the cyassl ?